public/kibi_radar_vis.js

```diff
@@ -12,65 +12,70 @@ import template from 'plugins/kibi_radar_vis/kibi_radar_vis.html';
 VisTypesRegistryProvider.register(RadarVisProvider);
 
 function RadarVisProvider(Private) {
-  const VisType = Private(VisVisTypeProvider);
-  const TemplateVisType = Private(TemplateVisTypeProvider);
+  const VisFactory = Private(VisFactoryProvider);
   const Schemas = Private(VisSchemasProvider);
 
   // return the visType object, which kibana will use to display and configure new
   // Vis object of this type.
-  return new TemplateVisType({
+  return VisFactory.createAngularVisualization({
     name: 'radar',
     title: 'Kibi Radar Chart',
     description: 'A radar chart is a graphical method of displaying multivariate data ' +
                  'in the form of a two-dimensional chart of three or more ' +
                  'quantitative variables represented on axes starting from the same point.' +
                  ' The relative position and angle of the axes is typically uninformative.',
     icon: 'fa-empire',
-    category: VisType.CATEGORY.KIBI,
+    category: CATEGORY.KIBI,
     template,
-    params: {
+    visConfig: {
       defaults: {
-        fontSize: 60,
-        shareYAxis: true,
-        addTooltip: true,
-        addLegend: true,
-        isDonut: false,
-        isFacet: false,
-        addLevel: true,
-        addAxe: true,
-        addVertice: true,
-        addPolygon: true,
-        addLevelLabel: true,
-        addAxeLabel: true,
-        addLevelScale: 1,
-        addLabelScale: 0.9,
-        addLevelNumber: 5
+        radar: {
```

Comment:
Why you introduced this wrapping object? Is this required by kibana6?

Reply:
Yes, apparently kibana 6 starts rendering its own legend after detecting these params.

Reply:
Can the name of the property be anything else instead of **radar**?
I mean does it have to match the **name** property?

Reply:
No, I just named it 'radar' as it's a radar vis. What name would you prefer?

Reply:
Radar is fine. I just wanted to know if it has to match the **name**
Actually, if you could just do the test to double check would be nice.
Like renaming it temporarily to something else like "radar1" and check that it would still work

Reply:
Sure, changed the variable to 'config' as it is somewhat descriptive.
It has come to my notice that you're not exactly using these configs:
fontSize
shareYAxis
addTooltip
isDonut
So i have removed them.